### DISCUSSION TITLE
Need help with Prototyping using Figma

### DISCUSSION THREAD

**Discussion by michaelg-roboyo** [primefaces/community#2214] [Figma UI Kit]
Hello everyone I was wondering how I can find tutorials or if Prime has their own engine for prototyping so I can use their components. Not everything has a sample prototype so if Prime does have their own design tool for prototyping please lmk I'm looking for options.

**Comment by onursenture** [comment]
Hello, I didn't quite understand what you meant by 'engine' for the prototype. The components in the UI Kit generally have prototype features, but there aren't any sample applications made with them if that's what you're looking for. If you are looking for a guide on prototyping, we recommend reading Figma's official guide: https://help.figma.com/hc/en-us/articles/360040314193-Guide-to-prototyping-in-Figma
